dev/TabView/TabView.xaml

```diff
@@ -43,20 +43,35 @@
         <Setter Property="Template">
             <Setter.Value>
                 <ControlTemplate TargetType="local:TabView">
-                    <Grid x:Name="TabViewContainer">
+                    <Grid>
+                        <VisualStateManager.VisualStateGroups>
+                            <VisualStateGroup x:Name="AddButtonStates">
```

Comment:
>AddButtonStates [](start = 54, length = 15)

Why are VSM states needed?  Can't you just bind AddButton.Visibility to IsAddButtonVisible?  There's default conversion between bool and visibility now, so that should just work.

Reply:
No kidding?? But when did that get added? We have to work downlevel to rs2. Sadly, I also tried it and it didn't work, not sure why; it builds without errors but Visibility="{TemplateBinding IsAddButtonVisible}" doesn't ever make it collapsed.

Reply:
It doesn't work with TemplateBinding, but should work with Binding RelativeSource=TemplatedParent

Reply:
That did work, although I am awfully sad we added something that fails invisibly like that.

Reply:
Maybe file another issue tracking that?  I agree that TemplateBinding should be at parity with Binding RelativeSource=TemplatedParent.

Reply:
OK, will do. (That and you can't use TemplateBinding in VisualState setters either...)